app/models/card.py

```diff
@@ -1 +1,30 @@
 from app import db
+from .board import Board
+from app.routes.routes_helpers import validate_model
+
+class Card(db.Model):
+    card_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    message = db.Column(db.String)
+    likes_count = db.Column(db.Integer)
+    board = db.relationship("Board", back_populates="cards")
+    board_id = db.Column(db.Integer, db.ForeignKey('board.board_id'))
+
+
+    def to_dict(self):
+        card_data = {
+            "card_id": self.card_id,
+            "message": self.message,
+            "likes_count": self.likes_count
+        }
+        if self.board:
+            card_data["board_id"] = self.board_id
```

Comment:
We conditionally included the goal id for task list since it was optional whether a task was part of a goal. Here, we expect all cards to belong to a board (this can even be enforced at a data level by requiring the board id foreign key to be non-null) so we could skip the conditional here.